.github/workflows/setup_perl_env.yml

```diff
@@ -4,26 +4,38 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Setup fastq-multx
+    - name: Checkout fastq-multx
       uses: actions/checkout@v2
       with:
         repository: 'brwnj/fastq-multx'
         path: fastq-multx
         ref: 'v1.4.0'
-    - run: |
-        mkdir -p $GITHUB_WORKSPACE/bin
+    - name: build and install fastq-multx
+      run: |
         cd fastq-multx
         make
-        cp fastq-multx $GITHUB_WORKSPACE/bin
-        echo "::add-path::$GITHUB_WORKSPACE/bin"
-        chmod +x $GITHUB_WORKSPACE/bin/fastq-multx
-        cd ..
-    - uses: actions/checkout@v2
-    - name: Setup perl & install htseq2multx
+        chmod +x fastq-multx
+        echo "::add-path::${GITHUB_WORKSPACE}/fastq-multx"
```

Comment:
Yeah, I wasn't sure I needed the mkdir and the cp, but I had tried adding the fastq-multx directory to the past using a relative path like in the docs, but it wasn't finding it.  When I found the `GITHUB_WORKSPACE` variable, I saw it in an example that used those mkdir/cp commands.  That example was using a container though.

So I'm wondering why this works... The significant differences of your changes are the htseq2multx build commands.  Do you think that's what did it?